### ISSUE TITLE
Content of switcher panel is announced even if panel is closed

### ISSUE THREAD

**Issue by joshblack** [carbon-design-system/carbon-website#1131]
Noticed this while using the website on Safari, it appears that the contents of the switcher are announced even if that panel is closed.

Steps to reproduce:

[panel-contents-announced.mov.zip](https://github.com/carbon-design-system/carbon-website/files/4540220/panel-contents-announced.mov.zip)
